src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets

```diff
@@ -135,4 +135,15 @@ Copyright (c) .NET Foundation. All rights reserved.
       </ReferenceCopyLocalPaths>
     </ItemGroup>
   </Target>
+
+  <Target Name="RemoveZipFileSuggestedRedirect"
```

Comment:
Please comment this. I know some poor souls are going to read this and wonder what it's about. This is bizarre enough that tests aren't enough convey "why" we're doing it.